app_dart/bin/server.dart

```diff
@@ -46,6 +46,17 @@ Future<void> main() async {
     );
 
     final Map<String, RequestHandler<dynamic>> handlers = <String, RequestHandler<dynamic>>{
+      /// Check+merge waiting github pull requests.
+      ///
+      /// Check if any github pull requests are marked with label
+      /// "waiting for tree to go green" and merge up to the number
+      /// specified within the handler.
+      ///
+      /// This api is called via cron job.
+      ///
+      /// GET: /api/check-waiting-pull-requests
+      ///
+      /// Response: Status 200 OK
       '/api/check-waiting-pull-requests': CheckForWaitingPullRequests(config, authProvider),
```

Comment:
Can we move these docs to class level dart docs? IDEs show class level docs, which makes it easy to surface this info here and in the tests.

Reply:
It would be nice for the docs to show up in the IDEs and stuff. I'm torn between having each section next to the call itself, which might make it easier to navigate, versus a massive block of text for the class.

Would having a class level doc saying, "go look at inline comments" be effective?

Reply:
ok, will move the whole block to class level after all feedback is incorporated.

Reply:
FWIW reviewing this PR would be easier if I could load the handlers inline

Reply:
Is great to have docs for all the different handlers. If refactoring is taking too much effort let's land this PR to add the docs and improve the docs in a different PR.